src/wp-includes/js/customize-preview-widgets.js

```diff
@@ -101,9 +117,9 @@ wp.customize.widgetsPreview = wp.customize.WidgetCustomizerPreview = (function(
 		},
 
 		/**
-		 * Refresh widget partial.
+		 * Refreshes the widget partial.
 		 *
-		 * @returns {Promise}
+		 * @returns {Promise|void} Either a promise postponing the refresh, or void.
```

Comment:
I am not sure `Promise|void` are the only return values, because we don't know what the `call` function will return on the `api.selectiveRefresh.Partial.prototype.refresh`.

Reply:
in file customize-selective-refresh, the refresh function also returns a promise. so it seems like it will always be either a promise or a void.